tests/unit/tasks/addon-install-test.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+var AddonInstallTask = require('../../../lib/tasks/addon-install');
+var MockProject      = require('../../helpers/mock-project');
+var expect           = require('chai').expect;
+var CoreObject       = require('core-object');
+var Promise          = require('rsvp').Promise;
+
+describe('addon install task', function() {
+  var ui;
+  var project;
+
+  beforeEach(function() {
+    ui = {
+      startProgress: function() {
+
+      }
+    };
+  });
+
+  afterEach(function() {
+    // ui.stopProgress();
+    ui = undefined;
+    project = undefined;
+  });
+
+  describe('when no save flag specified in blueprintOptions', function() {
+    it('calls npm install with --save-dev as a default', function() {
+      var mockNpmInstallTask = CoreObject.extend({
+        run: function(options) {
+          expect(options.save).to.not.equal(true);
+          expect(options['save-dev']).to.equal(true);
+          return new Promise(function(resolve, reject) {
+            resolve();
+          });
+        }
+      });
+
+      var addonInstallTask = new AddonInstallTask({
+        ui: ui,
+        project: project,
+        NpmInstallTask: mockNpmInstallTask
+      });
+
+      addonInstallTask.run({});
```

Comment:
I'm not convinced that this test will actually fail if things don't work as planned as it doesn't account for async behavior (there is neither a `done` callback, nor is a Promise returned by the test function)